src/OrbitGl/CallTreeView.h

```diff
@@ -154,6 +164,22 @@ class CallTreeUnwindErrors : public CallTreeNode {
   explicit CallTreeUnwindErrors(CallTreeNode* parent) : CallTreeNode{parent} {}
 };
 
+class CallTreeUnwindErrorNode : public CallTreeNode {
```

Comment:
Two things on the name: 
- the other classes don't end in "node", so this shouldn't either;
- `CallTreeUnwindError` is too similar to `CallTreeUnwindErrors`.

I think that this should be called `CallTreeUnwindErrorType` and all variables/fields/methods renamed accordingly.